CHANGELOG.md

```diff
@@ -4,9 +4,17 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### [v4.8.0-canary.0](https://github.com/microsoft/tabster/compare/v4.7.2...v4.8.0-canary.0)
+
+-   feat: Improving out of DOM order next focusables handling. [`d390d1d`](https://github.com/microsoft/tabster/commit/d390d1d777854db3e1aeac24c0ff1016519caeab)
+-   Oops. [`1a86249`](https://github.com/microsoft/tabster/commit/1a86249b50e157493ff20491fd90cf7d7026ee6a)
```

Comment:
Oops?

Reply:
That's not coming to the final merge. We undo the canary release commits and the squashed merge to master will only have the PR description.